src/diff/mod.rs

```diff
@@ -247,6 +247,19 @@ impl Diff {
         self.deleted.append(&mut new_files);
         Ok(())
     }
+
+    fn build_path(file: &File, parent_path: &Vec<Label>) -> Path {
+        match parent_path.len() {
+            0 => {
+                // TODO: Path::from_label ?
+                Path::from_labels(file.filename.clone(), &[])
+            },
+            _ => {
+                Path::from_labels(parent_path[0].clone(),
```

Comment:
Hmmmm this is making me think we should be passing around `Path` vs `Vec<Label>` even more :thinking: